VCFv4.4.draft.tex

```diff
@@ -523,6 +523,9 @@ \subsubsection{Genotype fields}
       \item $\mid$ : allele is phased (according to the phase-set indicated in {\tt PS} or {\tt PSL})
     \end{itemize}
 
+    For symbolic structural variant alleles, GT=0 indicates the absence of any of the ALT symbolic structural variants defined in the record.
```

Comment:
I think GT=0 should actually mean the absence of any structural variants for the region, even those no explicitly listed in the ALT column?

Reply:
It would be useful to define an unspecified structural variant ALT allele similar to the `<*>` that is used for gVCF ref blocks. This could be used both for copy neutral CNV records and for partially overlapping CNVs.

Reply:
@d-cameron One problem with defining GT=0 for SV events like this, is that it requires that ALT cannot be ".". If ALT is not "." then the definition of QUAL is changed, which is undesirable. It seems like using some kind of INFO field rather than the ALT column to inform interpretation of GT=0 would be preferable. I would say we should use SVTYPE for that, but it has been removed in VCF 4.4 right? Maybe this is justification to bring it back?

Reply:
SVTYPE was always redundant with ALT so there's no additional information conveyed by it.

Overall GT=0 is just a mess for SVs. If we define GT=0 as no other SVs (or SVs of the symbolic SV types defined in ALT)

The intent with going with the definition that I did was to match the existing semantics of SV callers. When an SV GT=0/1 claim they're claiming that the SV is heterozygous and, generally speaking, not making a claim about any other variant. This is slightly different to a CNV caller making a GT=0/1 claim as it's an implicit <CNV> CN=1 claim over SVLEN.

Complications for GT=0 are:
- There could be an earlier SV overlapping this position. An earlier <DUP> won't be listed in ALT for our position but it's definitely an SV that overlaps the interval
- Multiple SVLEN can be defined. It's unclear over what interval REF GT=0 is defined.
- BND claims have no interval - they're pointwise claims. GT=0 for these event types have a meaning closest to the usual REF GT meaning (although I don't know of any that actually validate that there's not a SNV/indel at POS).
- As defined, `<*>` means REF-only calls over the interval in which `<*>` is defined. SVs are not excluded so, as written, `<*>` technically excludes any symbolic SV calls starting in the interval and any breakpoints ending in the interval. I raised this a few months ago (#674) but there didn't see to be a need to clarify this at the time. This might need to be revisited.